.ci/check-bazel-version

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -euo pipefail
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# Ideally we would place .bazelversion in the top-level so it could be used by
+# tools like bazelisk. However, this breaks the nixpkgs provided Bazel
+# installation, see https://github.com/NixOS/nixpkgs/issues/80950.
+VERSION_EXPECTED="bazel $(cat "$DIR/bazelversion")"
+VERSION_ACTUAL=$(bazel version --gnu_format)
+# nixpkgs Bazel version ends on '- (@non-git)'.
```

Comment:
Should we fix this in nixpkgs?

Reply:
I think that would also fix [this issue](https://github.com/NixOS/nixpkgs/issues/80950#issuecomment-590337389), so that would be good. Though, I don't think we need to block this PR on it.

Reply:
I guess that’s already fixed by the changes in https://github.com/NixOS/nixpkgs/pull/80739#pullrequestreview-364046797

Reply:
IIUC that PR changes `bazel-real` for `bazel-VERSION...`, so it fixes https://github.com/NixOS/nixpkgs/issues/80950, but doesn't change the `- (@non-git)` part. (Unless Bazel 2.1 does that, I haven't checked).